src/Mod/Assembly/InitGui.py

```diff
@@ -51,8 +51,7 @@ class AssemblyWorkbench(Workbench):
     def __init__(self):
         print("Loading Assembly workbench...")
         self.__class__.Icon = (
-            FreeCAD.getResourceDir()
-            + "Mod/Assembly/Resources/icons/AssemblyWorkbench.svg"
+            FreeCAD.getResourceDir() + "Mod/Assembly/Resources/icons/AssemblyWorkbench.svg"
```

Comment:
Nice the 100 characters limit seems more forgiving than the standard black that Brad applied.

Reply:
We don't have a system in place to force the pre-commit hook (eventually we'll add a CI step that requires the formatting be correct, but that's not implemented right now). So actually installing pre-commit in the repo is a step that every developer must take.

Reply:
By the way regarding this pre-commit hook: It makes PR very hard to review as every file you touch get completely re-writen. I'm thinking about FlachyJoe's PR that were +-1500 lines when the real code was less than 100 lines.
So perhaps all files in the sources should be passed through a formating step now. Though that would make horrible conflicts to all the opened PR. Though the pre-commit hook only dilute the problem of making conflicts to other open PR.